paloesg/excide
# Send email to inform users for all tasks in unordered workflow

# Description
- Send email to inform ALL users involved in the unordered workflow about all the tasks they need to complete

Trello link: https://trello.com/c/K3m14hDo

## Remarks
- Nil

# Testing
- Created an unordered template to see the email notification sent. 
- Check if all user is sent ONCE
- Check with ordered template to see that the task notification is still working fine 

## Checklist:
